R/arc-select.R

```diff
@@ -79,9 +51,12 @@ arc_select <- function(
 
     if (any(!nindex)) {
 
-      stop(
-        "Field(s) not in `x`:\n  ",
-        paste(fields[!nindex], collapse = ", ")
+      cli::cli_abort(
+        paste0(
+          "Field(s) not in `x`:\n  ",
+          paste(fields[!nindex], collapse = ", ")
```

Comment:
I believe cli will auto-collapse and handle pluralization. Would you mind taking on pluralization here ?
This should be the trick here: 
```r
cli::cli_abort(
    "Field{?s} not in `x`: {.var {fields[!nindex]}}"
)
```